### PR TITLE
build: set up aarch64 cross-compilation toolchain

### PR DESCRIPTION
**Pull request by nvb**
This change configures the cockroachdb/builder docker image to be able
to target `aarch64-unknown-linux-gnueabi`. It does this by adding a new
crosstool-ng toolchain. This toolchain is almost identical to
`x86_64-unknown-linux-gnu` except that it targets the aarch64
architecture. It also uses the Linux 3.7.10 kernel instead of Linux
2.6.32 because Linux 3.7 was the first to support 64-bit ARM.

Release note (build change): The Docker builder image is now able to
cross-compile release aarch64 binaries.

### CODE REVIEW

File: build/builder.sh [hunk 1]
```diff
@@ -3,7 +3,9 @@
 set -euo pipefail
 
 image=cockroachdb/builder
-version=20180220-200046
+#version=20180220-200046
+# WIP update before merging.
+version=latest
```
Comment (benesch) [inline build/builder.sh]: don't forget about this!

**Review by benesch — APPROVED** [review]
LGTM modulo updating the builder!

File: build/builder.sh (hunk 1 above)
Comment (nvanbenschoten) [reply build/builder.sh]: Done.

**Comment by benesch** [comment]
If CI is happy so am I!

**Comment by nvanbenschoten** [comment]
TFTR!

**Pull request merged**